public/scripts/app.js

```diff
@@ -26,17 +26,17 @@ $(document).ready(function(){
   // make a GET request for all restaurants
   $.ajax({
     method: 'GET',
-    url: '/api/restaurant',
+    url: '/api/restaurants',
     success: handleSuccess,
     error: handleError
   });
 
-  $('#singlebutton').on('submit', function(e) {
+  $('#restaurant-form form').on('submit', function(e) {
     e.preventDefault();
     var formData = $(this).serialize();
 
-    $.post('/api/restaurant', formData, function(restaurant) {
-      renderAlbum(restaurant);
+    $.post('/api/restaurants', formData, function(restaurant) {
+      renderRes(restaurant);
```

Comment:
There is no `renderAlbum` function in this code. You need to use the `renderRes` function you created to render the restaurants.